python/ray/serve/config.py

```diff
@@ -113,6 +114,19 @@ def _needs_pickle(deployment_language: DeploymentLanguage, is_cross_language: bo
         return False
 
 
+class DeploymentOptionUpdateType(str, Enum):
+    # Nothing needs to be done other than setting the target state.
+    LightWeight = "LightWeight"
```

Comment:
let's name this something more descriptive. what options actually fit into this btw? is it basically `num_replicas` and so forth?

Reply:
It's `num_replicas` and `autoscaling_config`.